latest/src/app/documentation/demos/treetable/treetable.demo.ts

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {ClarityDocComponent} from "../clarity-doc";
+import {Component} from "@angular/core";
+
+const HTML_EXAMPLE_CLICKABLE_ROWS = `
+<clr-treetable>
```

Comment:
pls just show as much info in these examples as needed, to understand how it works. we don't need the whole copied code here.